Makefile

```diff
@@ -17,8 +17,10 @@ SD_VERSION   = 6.1.1
 SD_FILENAME  = $(SD_NAME)_nrf52_$(SD_VERSION)
 SD_HEX       = $(SD_PATH)/$(SD_FILENAME)_softdevice.hex
 
-# linker by MCU and SoftDevice eg. nrf52840_s140_v6.ld
-LD_FILE      = linker/$(MCU_SUB_VARIANT)_$(SD_NAME)_v$(word 1, $(subst ., ,$(SD_VERSION))).ld
+MBR_HEX			 = lib/softdevice/mbr/hex/mbr_nrf52_2.4.1_mbr.hex

```

Comment:
mbr hex from Nordic SDK v16 the verion is the same used by latest SD v6/v7

Reply:
can you please provide steps on how you got this file? I need to get such file for nrf5340.